gnowledge/gstudio
# view & template defined for partner.

separated out partner & group templates,views.
1. url defined for Partners 
<code>(r'^(?P<group_id>[^/]+)/ncert_partner', include('gnowsys_ndf.ndf.urls.partner'))</code>

2. <code>partner.html</code> template added for listing partners.

3. <code>create_partner </code> view defined for creating partner.

4. <code> create_partner.html </code> template added for creating new partner

<b>what to test:</b>
1. after selecting Partners  check url redirection
2. check partners listing view
3. try to create new partner by selecting "New Partner"

<b>to-do-list</b>
New Group becomes <code>New Partner</code> & NewSubGroup bocomes <code>NewSubPartner</code> in partner detail view
